internal/xds/translator/jsonpathtopointer_test.go

```diff
@@ -0,0 +1,374 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package translator
+
+import (
+	"sort"
+	"strconv"
+	"testing"
+
+	"github.com/ohler55/ojg/jp"
+	"github.com/stretchr/testify/require"
+)
+
+type pointerTest struct {
```

Comment:
to maintain uniformity, can these tests be represented in this https://github.com/envoyproxy/gateway/blob/eeb62c88f8949f8da8a1278ec5515ffa1a004444/internal/ir/xds_test.go#L533 format ? (moving the struct def inside the test itself)